jac-uk/apply
# Hotfix/1.64.2

## What's included?
Bug fix to make NINO fields optional rather than required.
Closes #993 

_Note: also includes hotfix 1.64.1 which doesn't appear to have been merged yet._

## Who should test?
✅ Product owner
✅ Developer

## How to test?
Visit each of the forms which include NINO field and check that it is now optional, though does validate the value if one is entered.
 - Create Account
 - Apply. Personal details
 - Character checks. Personal details

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
